plugins/modules/system/sysupgrade.py

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/python
+
+# Copyright: (c) 2020, Andrew Klaus <andrewklaus@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: sysupgrade
+short_description: Manage OpenBSD system upgrades
+description:
+    - Manage OpenBSD system upgrades using sysupgrade.
+options:
+    snapshot:
+        description:
+        - Apply the latest snapshot.
+        - Otherwise release will be applied.
+        default: no
+        type: bool
+    force:
+        description:
+        - Force upgrade (for snapshots only).
+        default: no
+        type: bool
+    keep_files:
+        description:
+        - Keep the files under /home/_sysupgrade.
+        - By default, the files will be deleted after the upgrade.
+        default: no
+        type: bool
+    fetch_only:
+        description:
+        - Fetch and verify files and create /bsd.upgrade but do not reboot.
```

Comment:
Maybe add more details here as well:
```suggestion
        - Fetch and verify files and create /bsd.upgrade but do not reboot.
        - Set to C(false) if you want sysupgrade to reboot. This will cause an error in Ansible, as Ansible does expect the module to exit. Also see the examples.
```

Reply:
I these changes look good. I've committed them.